cmd/bridge/main.go

```diff
@@ -457,7 +462,8 @@ func main() {
 			Endpoint:        k8sEndpoint,
 		}
 
-		if *fK8sModeOffClusterThanos != "" {
+		// Disable metrics in off-cluster multicluster env
+		if len(managedClusterConfigs) == 0 && *fK8sModeOffClusterThanos != "" {
```

Comment:
Shouldn't we disable it in any off-cluster scenario?
```suggestion
		if len(managedClusterConfigs) == 0 || *fK8sModeOffClusterThanos != "" {
```

Reply:
I think we still want metrics to work if an off-cluster thanos endpoint is provided through the `fK8sModeOffClusterThanos` flag.